legend-engine-authentication-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/authentication/OauthCredential.java

```diff
@@ -0,0 +1,8 @@
+package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.authentication;
+
+import org.finos.legend.engine.protocol.pure.v1.model.SourceInformation;
+public class OauthCredential extends Credential{
+
+    public String token;
```

Comment:
what's specific to oauth here?

Reply:
It just wraps the oauth token. I have removed this new class. We can resuse OAuthCredential from legend-engine-shared-core